charts/kubescape-operator/templates/_common.tpl

```diff
@@ -51,11 +51,11 @@ hostScanner:
 kubescape:
   enabled: {{ eq .Values.capabilities.configurationScan "enable" }}
 kubescapeScheduler:
-  enabled: {{ and $configurations.submit (eq .Values.capabilities.configurationScan "enable") }}
+  enabled: {{ eq .Values.capabilities.configurationScan "enable" }}
```

Comment:
have you tried running a cluster like that? if I remember correctly we had issues running with both continuousScan and scheduler